tensionmap.py

```diff
@@ -221,6 +244,12 @@ def tm_update_selected(self, context):
     :param context: the context in which the selected object is
     :return: nothing
     """
+    global object_data
+    if context.object.data.tm_active:
+        object_data[context.object.name]=Object_Data(context.object)
+    else:
+        del object_data[context.object.name]
```

Comment:
Since you are using `context.object.name` no matter what, putting it in a variable can make the code more readable, and depending on how Python and the Blender API work, maybe also a bit faster.

`context.object` could also be put in a variable. I'm okay with either options here.

Reply:
Done!